packages/main/src/plugin/color-registry.ts

```diff
@@ -322,6 +322,11 @@ export class ColorRegistry {
       light: colorPalette.gray[50],
     });
 
+    this.registerColor('titlebar-windows-bg', {
+      dark: '#202020',
+      light: colorPalette.gray[50],
+    });
```

Comment:
shouldn't we set these values into `titlebar-bg`

but adding a condition like
```
dark : isWindows() ? '#202020' : colorPalette.something
```

so there is only one definition of the color, just that the default value change based on the current Operating System
the svelte component just read the value

Reply:
@benoitf Like so?